rpcd/playbooks/roles/rpc_maas/templates/disk_utilisation.yaml.j2

```diff
@@ -1,8 +1,10 @@
+{% set custom_maas_check_period = maas_check_period_override | selectattr('label', 'equalto', 'disk_utilisation') | map(attribute='value') %}
```

Comment:
I wish there was, @nrb. :( I wish I could use dictionary comprehension within a Jinja2 template as this would be much cleaner.